cmd/podman/logout.go

```diff
@@ -65,13 +66,26 @@ func logoutCmd(c *cliconfig.LogoutValues) error {
 		return nil
 	}
 
-	err := config.RemoveAuthentication(sc, server)
-	switch err {
+	// username of user logged in to server (if one exists)
+	userFromAuthFile, passFromAuthFile, err := config.GetAuthentication(sc, server)
+	if err != nil {
+		return errors.Wrapf(err, "error reading auth file")
+	}
+
+	islogin := docker.CheckAuth(getContext(), sc, userFromAuthFile, passFromAuthFile, server)
+
+	err = config.RemoveAuthentication(sc, server)
```

Comment:
Isn't this attempting to remove the authentication from Docker's cache and not Podman's?

Reply:
this only changes the podman auth.json